docs/root/modules/lastpass/config.md

```diff
@@ -0,0 +1,9 @@
+## Lastpass Configuration
+
+.. _lastpass_config:
+
+Follow these steps to analyze Lastpass objects with Cartography.
+
+1. Prepare your Lastpass CID & ProvHash key.
+    1. Get your CID (account number) and ProvHash from Lastpass [Where can I find the CID and API secret?](https://support.lastpass.com/help/where-can-i-find-the-cid-and-api-secret)
```

Comment:
Curious, what privileges does this provhash grant? Is the authz configurable?

Reply:
Sadly it's Lastpass ... so no configuration/restriction (do not judge me, Lastpass is a Legacy app, working on new solution ;) )

The API is mainly designed to allow provisioning (add/edit/delet user & groups)
You can add/access enterprise level shared access
Thanksfully you can not access user's secrets